apps/wing-console/console/server/src/index.ts

```diff
@@ -45,6 +45,8 @@ export type {
   LayoutComponentType,
 } from "./utils/createRouter.js";
 
+export * from "@winglang/sdk/lib/ex/index.js";
```

Comment:
Is this needed?

Reply:
Currently, yes. We're bundling the server files into a single `index.js` and `index.d.ts` file, and the bundler can't generate the `index.d.ts` properly without exporting all the types the server exposes indirectly.

Worth investigating if we can get along without bundling because we already have a lot of these kind of exports in the package.